### PR TITLE
The new structure of `Kolibri-windows-installer` repo

### PR DESCRIPTION
**Pull request by mrpau-julius**
### SUMMARY
I enhance the structure and files of the Kolibri-windows-installer repo. I rename the windows to src and remove/merge the duplicate files.

REF: #105 

### Notes
WIP - Don't merge yet

### CODE REVIEW

**Comment by benjaoming** [comment]
Hi Julius :) The `.gitignore` should remain in the root directory, but you might have to adjust some of the paths defined inside the file.

**Comment by benjaoming** [comment]
It would also be great if this project only has 1 single README.md file, right now it has two.. both in the root and in the `windows/` directory. This should only be found in the root.

**Comment by mrpau-julius** [comment]
Thank you for the comment @benjaoming. Today I will move the `.gitignore`, `license` and `README.md` to the root.

**Comment by indirectlylit** [comment]
adding milestone 1.3.2

**Comment by indirectlylit** [comment]
looks like this work might need to be rebased

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,33 @@
...
+*.suo
+
+# Compiled executable
+!src/*.exe
+
+# Dynamic link library
+!src/*.dll
+
+# KA Lite source folder
+ka-lite/
```
Comment (indirectlylit) [inline .gitignore]: this shouldn't exist! :)

Comment (mrpau-julius) [reply .gitignore]: Thanks, @indirectlylit for reviewing, I will remove that today.

**Comment by mrpau-julius** [comment]
@mrpau-richard please review my PR. :smiley:

**Comment by mrpau-richard** [comment]
Looks good to me. merging this.

**Pull request merged**